config/default.js

```diff
@@ -93,9 +93,7 @@ var config = {
       create: true
     }
   ],
-  dist: 'dist',
-  lambdaSrcModule: 'particles-cloudsploit-scans',
-  lambdaSrc: 'cloudsploit-scans'
```

Comment:
In order to avoid hard coding of the lambda source lambdaSrc config parameter was added, think we need it.